src/model/ai/evaluation/BackRowCountEvaluator.java

```diff
@@ -22,18 +21,25 @@ public double evaluateBoard(Board theBoard, PieceColor color) {
         for (Square square : theBoard.getGameState()) {
```

Comment:
I don't think we should cycle through all of the squares on the board in this Evaluator. We should just check squares 1-4 for black pieces and 29-32 for white pieces. Also, there probably should be some added bonus for a completely intact back row. Finally, this should look at the current player's color, and add points for an intact back row for itself, but I don't think it should look at the other player's back row, and if it does, the counting of points should be inverse, such that an intact back row for your opponent is a bad thing.